readme/testing.md

```diff
@@ -75,13 +75,17 @@ Each testing type can be either executed directly, or via a corresponding Phing
 * `blt tests:behat`
 * `blt tests:phpunit`
 
+To execute a single test (with Phing):
+
+* `blt tests:behat:execute -Dbehat.param=tests/behat/features/Examples.feature`
```

Comment:
Are you sure this works? I don't see how behat.param is passed to the call to Behat in Phing.